### PR TITLE
DAOS-8915 build: Build and test on Leap 15.3

### PR DESCRIPTION
**Pull request by brianjmurrell**
Remove build stages duplicted in GHA to make more room in the Jenkinsfile
for other required stages.

Force Leap RPM test stage to use 15.2 so that we have coverage for that
platform as we move main testing to Leap 15.3.

### CODE REVIEW

**Review by JohnMalmberg — APPROVED** [review]

File: utils/rpms/packaging/Dockerfile.mockbuild [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-# Copyright 2018-2020, Intel Corporation
+# Copyright 2018-2021, Intel Corporation
 #
 # 'recipe' for Docker to build an RPM
 #
 
 # Pull base image
 FROM fedora:latest
-MAINTAINER daos-stack <daos@daos.groups.io>
+LABEL maintainer="daos@daos.groups.io>"
```
Comment (phender) [inline utils/rpms/packaging/Dockerfile.mockbuild:9]: Did we miss removing the `>`?
 ```suggestion
LABEL maintainer="daos@daos.groups.io"
```

**Review by phender — CHANGES_REQUESTED** [review]

File: utils/rpms/packaging/Dockerfile.mockbuild (hunk 1 above)
Comment (brianjmurrell) [reply utils/rpms/packaging/Dockerfile.mockbuild:9]: Fixed in next push

**Review by phender — APPROVED** [review]

**Review by JohnMalmberg — APPROVED** [review]

File: utils/rpms/packaging/Dockerfile.coverity [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2019, Intel Corporation
+# Copyright 2018-2020, Intel Corporation
```
Comment (ashleypittman) [inline utils/rpms/packaging/Dockerfile.coverity:2]: Do you need to update this, and there's a permissions change on this file.

File: utils/rpms/packaging/Dockerfile.ubuntu.20.04 [hunk 3]
```diff
@@ -1,11 +1,11 @@
 #
-# Copyright 2019-2020, Intel Corporation
+# Copyright 2019-2021, Intel Corporation
```
Comment (ashleypittman) [inline utils/rpms/packaging/Dockerfile.ubuntu.20.04:2]: Still no CACHEBUST in this file?

**Comment by brianjmurrell** [comment]
Failed on known issue [DAOS-9038](https://daosio.atlassian.net/browse/DAOS-9038) (which is a suspected duplicate of [DAOS-8901](https://daosio.atlassian.net/browse/DAOS-8901)).

File: utils/rpms/packaging/Dockerfile.ubuntu.20.04 (hunk 3 above)
Comment (brianjmurrell) [reply utils/rpms/packaging/Dockerfile.ubuntu.20.04:2]: That's an issue for daos-stack/packaging, which should in fact be a submodule here.

File: utils/rpms/packaging/Dockerfile.coverity (hunk 2 above)
Comment (brianjmurrell) [reply utils/rpms/packaging/Dockerfile.coverity:2]: That's an issue for daos-stack/packaging which is just being copied to here from there.

**Pull request merged**